README.md

```diff
@@ -606,6 +606,83 @@ This method will continuously execute cairo steps until the end pc, returned by
     //TODO
 ```
 
+*Step*
+
+```go
+    //TODO
+```
+
+*Decode instruction*
+
+```go
+    //TODO
+```
+
+*Run instruction*
+
+```go
+    //TODO
+```
+
+*Compute operands*
+
+```go
+    //TODO
+```
+
+*Opcode assertions*
+
+Once we have the instruction's operands to work with, we have to ensure the correctness of them. The first thing we need to differenciate is which type of instruction are we running, we do this by switching the `opcode`. 
+
+The posible options (for now) are: 
+	1. AssertEq instruction 
+	2. Call instruction 
+
+In the first option, we need to ensure the result operand is not null (nil in this case) and also that the result operand is equal to the destiny operand. If any of those things fail, we throw an error. 
+
+On the other hand, the Call instruction, what we do first is define our return PC register, we do that adding the size of the instruction to the current PC. Then, we check our operand OP0 is equal to the return PC and our destiny operand is the same as the return FP register. If any of those things fail, we throw an error. 
```

Comment:
```suggestion
On the other hand, the Call instruction, what we do first is define our return PC register, we do that adding the size of the instruction to the current PC. Then, we check our operand OP0 is equal to the return PC and our destiny operand is the same as the return FP register. If any of those things fail, we throw an error. 
```
destiny operand doesn't sound quite right, we should use destination, or just dst instead

Reply:
I agree!